short fixes for cut, crush, bandq

bgold-cosmos opened the pull request:
SuperDirt internally uses “~cutgroup” but the Tidal keyword by default
is “cut”, so I added a translation. Similarly, SuperDirt uses “bandqf”
whereas the Tidal parameter is “bandf”. Finally, I made a change to
crush to match expected quantization to the specified number of bits.

I can separate these out into different pull requests if desired, but they're
all so small it didn't seem worth it.

telephon commented:
>I'm not sure what those merge patches are doing, hopefully I haven't broken anything..

I also always wonder why it is so hard to maintain one's own branch and then commit back to the origin. As you are member of musikinformatik, you should be able to directly push changes, or a topic branch that we can merge.

Commit:
short fixes for cut, crush, bandq

SuperDirt internally uses “~cutgroup” but the Tidal keyword by default
is “cut”, so I added a translation. Similarly, SuperDirt uses “bandqf”
whereas the Tidal parameter is “bandf”. Finally, I made a change to
crush to match expected quantization to the specified number of bits.

telephon commented:
thank you!
